cmd/networking/ns/main.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This new directory (see comments below about what it's called) needs `kodata` like our other controllers that symlink various things.